ui/src/app/views/workspace-share/component.css

```diff
@@ -1,6 +1,5 @@
 input.share, .sharing-buttons {
```

Comment:
Do you have control over the padding/margin on the overall modal content? It would be great if that could be reduced.

Reply:
i.e. the whitespace between the top of the modal and the modal title (see the mocks for comparison).

Reply:
Let me see what I can do.